packages/env/README.md

```diff
@@ -188,6 +188,20 @@ wp-env start --debug
 	...
 ```
 
+## Using included WordPress PHPUnit test files
+
+Out of the box `wp-env` includes the [WordPress' PHPUnit test files](https://develop.svn.wordpress.org/trunk/tests/phpunit/) corresponding to the version of WordPress installed. There is an environment variable, `WP_TESTS_DIR`, which points to the location of these files within each container. By including these files in the environment, we remove the need for you to use a package or install and mount them yourself.
+
+### Custom `wp-tests-config.php` file
+
+WordPress provides a `WP_TESTS_CONFIG_FILE_PATH` constant that you can use to change the `wp-config.php` file used for testing. Set this to a desired path in your `bootstrap.php` file and the file you've chosen will be used instead of the one included in the environment.
+
+### Using `wp-phpunit/wp-phpunit`
```

Comment:
Honestly, I don't know if we need to document this explicitly. I see the value of mentioning `WP_TESTS_DIR` and the bootstrap.php file, but I wonder if we really need to mention that wp-phpunit can still be used if our main goal is to discourage it in favor of the built-in solution

Reply:
I guess it depends on whether or not you feel like we've explicitly supported the `wp-phpunit/wp-phpunit` use case. I suppose it's good to remove it, there are likely two kinds of users of the `wp-phpunit/wp-phpunit` package for us:

* Those who include the package but rely 100% on `wp-env` to bootstrap and use it. These would be unaffected by the change and it would be a silent transition.
* Those who include the package and support custom environments. These would be unaffected by the change since they need to set some of these variables in the first place. The one thing worth mentioning is the `WP_TESTS_DIR` bit.

With that in mind, I've gone ahead and removed the section and extracted the only useful bits.